packages/react-select/src/Select.js

```diff
@@ -1420,9 +1420,12 @@ export default class Select extends Component<Props, State> {
     const { inputIsHidden } = this.state;
 
     const id = inputId || this.getElementId('input');
+    const listBoxId = this.getElementId('listbox');
 
     // aria attributes makes the JSX "noisy", separated for clarity
     const ariaAttributes = {
+      role: 'combobox',
```

Comment:
I believe this should be `'combobox'` if it's searchable, but `'listbox'` otherwise; however, I'm not certain what that means for other attributes (re: `'aria-owns'`).

Reply:
Aria owns is for searchbox, the list that it owns